src/main/scala/cromwell/backend/BackendValidationActor.scala

```diff
@@ -0,0 +1,53 @@
+package cromwell.backend
+
+import akka.actor.{Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendValidationActor.{Validate, ValidationResult}
+import wdl4s.{WorkflowCoercedInputs, NamespaceWithWorkflow}
+
+import scala.concurrent.Future
+
+object BackendValidationActor {
+  sealed trait BackendValidationActorMessage
+  case class Validate(namespace: NamespaceWithWorkflow, wfInputs: Option[WorkflowCoercedInputs] = None, wfOptionsJson: Option[String] = None) extends BackendValidationActorMessage
```

Comment:
Could this be final?

Also I generally advocate that options in these situations really should be split out into individual components of the sum type although with more than option like here I'll admit it gets messy (you'd need 4 separate classes here)